test/expr/unary/selector/selector.swift

```diff
@@ -116,7 +116,7 @@ func testParseErrors3(_ c1: C1) {
 
 func testParseErrors4() {
   // Subscripts
-  _ = #selector(C1.subscript) // expected-error{{type 'C1.Type' has no subscript members}}
+  _ = #selector(C1.subscript) // expected-error{{type 'C1' has no member 'subscript'}}
```

Comment:
I think we *are* going to want to support syntax like this for getting the selector of a subscript getter or setter some day: `#selector(getter:C1.subscript)`. It doesn't currently work but that doesn't mean it *shouldn't* work.